simpl/config.py

```diff
@@ -477,8 +477,15 @@ def parse_cli(self, argv=None, permissive=False):
         options = []
         for option in self._options:
             kwargs = option.kwargs.copy()
-            if kwargs.get('default') is None:
```

Comment:
`argparse.SUPPRESS is not None` - so the deleted line handled `default=argparse.SUPPRESS` correctly, I think and the new one may miss it?